src/test/java/com/redhat/devtools/lsp4ij/features/completion/CssCompletionClientConfigTest.java

```diff
@@ -0,0 +1,445 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij.features.completion;
+
+import com.redhat.devtools.lsp4ij.fixtures.LSPCompletionClientConfigFixtureTestCase;
+
+/**
+ * Tests code completion client configuration settings using a mock CSS language server.
+ */
+public class CssCompletionClientConfigTest extends LSPCompletionClientConfigFixtureTestCase {
```

Comment:
The CSS test exercises whitespace-delimited argument list snippets as well as some complex snippets with variable values that are enumerated lists/choices.